app/actions/ControlActions.js

```diff
@@ -348,8 +348,10 @@ export const purchaseTicketsAttempt = (
         );
       }
       accountNum = account.encrypted ? account.value : null;
+      // Add default account unlock if account isn't default
```

Comment:
I suggest adding the reason for adding the default account here.

Something to the effect of "... because the voting address will be derived from the default account".